sycl/plugins/opencl/CMakeLists.txt

```
@@ -28,9 +23,9 @@ set_target_properties(pi_opencl PROPERTIES LINKER_LANGUAGE CXX)
 target_include_directories(pi_opencl PRIVATE "${sycl_inc_dir}")
 
 #link pi_opencl with OpenCL headers and ICD Loader.
-target_link_libraries( pi_opencl
-  PRIVATE OpenCL::Headers
-  PRIVATE ${OpenCL_LIBRARIES}
+target_link_libraries(pi_opencl
+  PRIVATE OpenCL-Headers
+  PRIVATE OpenCL-ICD
```

```suggestion
  PRIVATE
    OpenCL-Headers
    OpenCL-ICD
```
To align code style with the rest of the project CMake files.